docs/content/users/quickstart.md

```diff
@@ -375,6 +375,31 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
     For more advanced tasks like adding elasticsearch, building and watching storefront and administration, see [susi.dev](https://susi.dev/ddev-shopware-6).
 
+=== "Silverstripe"
+
+    ## Silverstripe
+
+    === "Silverstripe"
+
+        Use a new or existing Composer project, or clone a Git repository.
+
+        === "Composer"
+            ```bash
+            mkdir my-silverstripe-app
+            cd my-silverstripe-app
+            ddev config --project-type=silverstripe --docroot=public --create-docroot
+            ddev composer create --prefer-dist --no-scripts silverstripe/installer -y
+            ddev start
+            ```
+        === "Git Clone"
+            ```bash
+            git clone <your-silverstripe-repo>
+            cd <your-silverstripe-project>
+            ddev config --project-type=silverstripe --docroot=public --create-docroot
+            ddev start
+            ddev composer install
+            ```
+
```

Comment:
Since the changes that are applied in the .env file are not user facing and could go unnoticed i would suggest to explicitly name the username and password you are able to log into silverstripe on the login page that was launched with `ddev launch /admin`. that way it would be clear to the user how to proceed. with the env changes a default user is in place now but still it would be not directly clear to the user what the necessary credentials are and that those are already added to the .env file. i would suggest something like similar to moodle: 

```
Log into the Silverstripe site you just installed using `admin` and `password`. Visit [Learn Silverstripe](https://www.silverstripe.org/learn/) for more information.